src/components/modals/ManagePDNTModal/ManagePDNTModal.tsx

```diff
@@ -62,6 +41,28 @@ function ManagePDNTModal() {
   const [modifiedValue, setModifiedValue] = useState<string | number>();
   const [rows, setRows] = useState<ManagePDNTRow[]>([]);
   const [loading, setLoading] = useState<boolean>(true);
+  const [showTransferPDNTModal, setShowTransferPDNTModal] =
+    useState<boolean>(false);
+
+  const EDITABLE_FIELDS = [
+    'name',
+    'ticker',
+    'targetID',
+    'ttlSeconds',
+    'controller',
+  ];
+
+  const ACTION_FIELDS: {
+    [x: string]: {
+      title: string;
+      fn?: () => void;
+    };
+  } = {
+    owner: {
+      title: INTERACTION_TYPES.TRANSFER,
+      fn: () => setShowTransferPDNTModal(true),
+    },
```

Comment:
i think we can just remove this and do an if check below

Reply:
added